recipes/neuralforecast/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "neuralforecast" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/neuralforecast-{{ version }}.tar.gz
+  sha256: 31e27969195f03b9ccfb6cab39b339c8c1d52d4048ce727d3a6ffab1dceb2e94
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - fastcore
+    - gdown
+    - hyperopt
+    - matplotlib-base
+    - numba
+    - openpyxl
+    - pandas
+    - pip
+    - py7zr >=0.18.1
```

Comment:
I cannot find that version specifier upstream. is that from your pip checks?

Reply:
Yes, pip check was showing that. I am quite sure, if pip check is dropped, that version `>=0.18.1` will not be necessary.